catalog-rest-service/src/main/resources/json/schema/configuration/kafkaEventConfiguration.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/configuration/kafkaEventConfiguration.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "KafkaEventConfiguration",
+  "description": "This schema defines the Kafka Event Publisher Configuration.",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.kafka.KafkaEventConfiguration",
+  "properties": {
+    "topics": {
+      "description": "Topics of Kafka Producer",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "acks": {
+      "description": "Acknowledgment",
+      "type": "string"
+    },
+    "retries": {
+      "description": "No. of retries",
+      "type": "integer"
+    },
+    "lingerMS": {
+      "description": "Artificial Delay",
+      "type": "integer"
+    },
+    "bufferMemory": {
+      "description": "Buffer Memory",
+      "default": 33554432,
+      "type": "integer"
+    },
+    "keySerializer": {
+      "description": "Serializer class for key",
+      "type": "string"
+    },
+    "valueSerializer": {
+      "description": "Serializer class for value",
+      "type": "string"
+    }
+  },
+  "required": ["topics", "keySerializer", "valueSerializer"],
```

Comment:
topic is only required , for other two serialised we can add default value of *.StringSerializer that we had saw earlier

Reply:
we can have opinionated instantiation , with default value for most of the properties , in json give default properties rest of the things that are mandatory like topic should be added to required

Reply:
Addressed in the latest commit.